commons/src/main/res/values-bs/strings.xml

```diff
@@ -0,0 +1,1205 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you translated this file?

Reply:
Yes, I mean most of the strings are the same as in Croatian (`values-hr`), with some minor differences.